### PR TITLE
It2 percentage home wins and percentage_ties

### LINKED ISSUE #13: Game stats - percentage home wins
Description:
Percentage of games that a home team has won (rounded to the nearest 100th)

Return value:
Float

### LINKED ISSUE #15: Game stats - percentage ties
Description:
Percentage of games that has resulted in a tie (rounded to the nearest 100th)

Return value:
Float

### PR DESCRIPTION
**Pull request by AngelaGuardia**
**Changes Made**
- Updated the StatTrackerTest to use a class variable instead of a setup method
- Cleaned up StatTracker attr_reader and initialize methods (with corresponding tests)

**Additions**
- Added total_games, total_home_wins, total_tied_games helper methods (with corresponding tests)
- Added percentage_home_wins and percentage_ties methods (with corresponding tests)

### CODE REVIEW

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -3,48 +3,67 @@
...
-    @stat_tracker = StatTracker.from_csv(@fixture_locations)
-  end
+    @@stat_tracker = StatTracker.from_csv(fixture_locations)
 
   def test_it_exists
-    assert_instance_of StatTracker, @stat_tracker
+    assert_instance_of StatTracker, @@stat_tracker
+  end
+
+  def test_it_has_attributes
```
Comment (sarynm12) [inline test/stat_tracker_test.rb:22]: Looks good! Thank you for cleaning this up!

**Review by sarynm12 — APPROVED** [review]
This all looks really good, Angela! Thank you!

**Review by leahriffell — APPROVED** [review]
Thank you for adding in all the collections of all of our fixture files! The clean up is great as well. I like having helper methods live in one group vs. broken out by stat type. 

Your helper and stat methods also look good to me.

**Review by hopesgit — APPROVED** [review]

**Comment by sarynm12** [comment]
Really good, Angela! Nice work! If everyone feels good, I'll go ahead and merge

**Pull request merged**